features/plugin-activate.feature

```diff
@@ -62,6 +62,27 @@ Feature: Activate WordPress plugins
       Plugin 'user-switching' activated.
       """
 
+  Scenario: Activating a plugin with no network wide option passes down correct types
```

Comment:
Not sure if there is a better `.feature` file to place this test into. It's rather too specific and sticks out from the other feature tests.

Reply:
I think this should be fine. Thanks for flagging 😊